Tests/Foundation/Tests/TestAffineTransform.swift

```diff
@@ -7,383 +7,741 @@
+            file: file, line: line
+        )
+        XCTAssertEqual(
+            nsTransform.transform(size), newSize,
+            "Expected NSAffineTransform to match AffineTransform's size-accepting transform(_:)",
+            file: file, line: line
+        )
+        
+        XCTAssertEqual(
+            newPoint.x, expectedPoint.x,
+            accuracy: accuracyThreshold,
+            "Invalid x: \(message)",
+            file: file, line: line
+        )
+        
+        XCTAssertEqual(
+            newPoint.y, expectedPoint.y,
+            accuracy: accuracyThreshold,
+            "Invalid y: \(message)",
+            file: file, line: line
+        )
     }
+}
 
-    func test_BasicConstruction() {
-        let identityTransform = NSAffineTransform()
-        let transformStruct = identityTransform.transformStruct
-
-        // The diagonal entries (1,1) and (2,2) of the identity matrix are ones. The other entries are zeros.
-        // TODO: These should use DBL_MAX but it's not available as part of Glibc on Linux
-        XCTAssertEqual(Double(transformStruct.m11), Double(1), accuracy: accuracyThreshold)
-        XCTAssertEqual(Double(transformStruct.m22), Double(1), accuracy: accuracyThreshold)
-
-        XCTAssertEqual(Double(transformStruct.m12), Double(0), accuracy: accuracyThreshold)
-        XCTAssertEqual(Double(transformStruct.m21), Double(0), accuracy: accuracyThreshold)
-        XCTAssertEqual(Double(transformStruct.tX), Double(0), accuracy: accuracyThreshold)
-        XCTAssertEqual(Double(transformStruct.tY), Double(0), accuracy: accuracyThreshold)
+// MARK: - Construction
+
+extension TestAffineTransform {
+    func testConstruction() {
+        let transform = AffineTransform(
+            m11: 1, m12: 2,
+            m21: 3, m22: 4,
+             tX: 5,  tY: 6
+        )
+        
+        XCTAssertEqual(transform.m11, 1)
+        XCTAssertEqual(transform.m12, 2)
+        XCTAssertEqual(transform.m21, 3)
+        XCTAssertEqual(transform.m22, 4)
+        XCTAssertEqual(transform.tX , 5)
+        XCTAssertEqual(transform.tY , 6)
     }
+}
 
-    func test_IdentityTransformation() {
-        let identityTransform = NSAffineTransform()
+// MARK: - Bridging
 
-        func checkIdentityPointTransformation(_ point: NSPoint) {
-            checkPointTransformation(identityTransform, point: point, expectedPoint: point)
-        }
+extension TestAffineTransform {
+    func testBridging() {
+        let transform = AffineTransform(
+            m11: 1, m12: 2,
+            m21: 3, m22: 4,
+             tX: 5,  tY: 6
+        )
         
-        checkIdentityPointTransformation(NSPoint.zero)
-        checkIdentityPointTransformation(NSPoint(x: 24.5, y: 10.0))
-        checkIdentityPointTransformation(NSPoint(x: -7.5, y: 2.0))
+        let nsTransform = NSAffineTransform(transform: transform)
+        XCTAssertEqual(transform, nsTransform.affineTransform)
```

Comment:
`.affineTransform` is an `internal` property so will need to be gated with `NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT` - there are a few uses of this which will also require it.